src/module.ts

```diff
@@ -0,0 +1,157 @@
+import { PanelConfig } from './panel-config';
```

Comment:
Split your class (too many features: creating diagram, rest calls, config etc...)

Reply:
done. I created a ac2-map module to handle the gojs diagram part